src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -31,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  setTodos: (todos: Todo[]) => void;
+  todos: Todo[] | undefined;
+}
+
+export const TodoFilter: React.FC<Props> = ({ setTodos, todos }) => {
+  const [select, setSelect] = useState('all');
```

Comment:
Consider using a more descriptive variable name for `select`, such as `filterStatus`, to improve code readability and make it clear what this state represents.